### ISSUE TITLE
Running code from python files.

### ISSUE THREAD

**Issue by GrandMoff100** [spacemanspiff2007/sphinx-exec-code#1]
I understand that this plugin runs python code that is hard coded into the documentation. But in my use case I have example python files implementing my project that I would like run dynamically when building the documentation. Similar plugins have the option of doing something like the following:
```rst
.. exec-code::
   :filename: examples/hex_codes.py
   :linenos:
```

Do you think you could implement this for this project?
Thanks!

**Comment by spacemanspiff2007** [comment]
Available in 0.4.

You can also set the path to the example folder so you don't have to specify it with every ``exec-code`` directive.

**Comment by GrandMoff100** [comment]
It works perecectly! Thank you!
